thoth/storages/graph/dgraph.py

```diff
@@ -763,6 +763,22 @@ def retrieve_unsolved_python_packages(self, solver_name: str) -> dict:
 
         return self._postprocess_retrieve_packages(result)
 
+    def retrieve_not_analyzed_python_package_version_entities(self) -> List[dict]:
+        """Retrieve a list of package names, versions and index urls that were not analyzed yet by package-analyzer."""
+        query = """{
+            f(func: has(%s), first: 10) @filter(NOT has(%s)) {
+                package_name:package_name
```

Comment:
I think just adding `package_name` instead of `package_name:package_name` should be enough, but I looked at how other queries are implemented and followed the convention.

Reply:
aliases are good when `@normalize` is used.